src/main/java/com/marklogic/appdeployer/command/AbstractCommand.java

```diff
@@ -104,7 +104,7 @@ public void setResourceFilenamesIncludePattern(Pattern pattern) {
      */
     protected String copyFileToString(File f) {
         try {
-            return new String(FileCopyUtils.copyToByteArray(f));
+            return new String(FileCopyUtils.copyToByteArray(f.getAbsoluteFile()));
```

Comment:
Any thoughts on how a JUnit test would verify that this fixes something?

Reply:
I will try to create a unit test later this day or tomorrow.
Not sure how to change/fake the current working directory to get this done thought.